### DISCUSSION TITLE
apisix install failed in docker desktop

### DISCUSSION THREAD

**Discussion by NICEXAI** [apache/apisix#4910] [Q&A]
I failed to install apisix inside docker desktop on windows because I have XunLei download tool installed on my computer. DownloadSDKServer.exe in Xunlei has the same port 9080, which eventually causes the port to be occupied. Will apisix consider changing the default port to a less common port, e.g., 2333?

**Comment by tokers** [comment]
@NICEXAI You can use another port by mounting the configuration file to the container.
